src/main/webapp/index.xhtml

```diff
@@ -1,56 +1,58 @@
-                <h:commandButton class="barcode-input-btn active" 
-                                 value="Confirm"
-                                 action="#{localTradeList.submit}">
-                    <!--ajax for async update of table -->
-                    <f:ajax execute="form" resetValues="true" render=":trade-list errmsg limit-err" />
-                </h:commandButton>
-                <h:message id="errmsg" class="error" for="bcode"/>
-
-            </h:form>
-            <!-- dataTable instead of ui:repeat for clear and easy table -->
-            <h:dataTable id="trade-list" 
-                         value="#{localTradeList.items}" 
-                         var="item"
-                         styleClass="trade-table hidden"
-                         style="display:#{localTradeList.items.isEmpty() ? 'none;' : 'block'}">
-                <h:column>
-                    <f:facet name="header">Barcode</f:facet>
-                    #{item.barcode}
-                </h:column>
-                <h:column>
-                    <f:facet name="header">Item</f:facet>
-                    #{item.name}
-                </h:column>
-            </h:dataTable>
-        </h:panelGroup>   
-        
-    </h:body>
-</html>
-
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:h="http://xmlns.jcp.org/jsf/html"
+      xmlns:pt="http://xmlns.jcp.org/jsf/passthrough"
+      xmlns:f="http://xmlns.jcp.org/jsf/core">
+    <h:head>
+        <meta charset="utf-8" />
+        <h:outputStylesheet name="css/style.css" />
+        <title>Fizzit</title>
+    </h:head>
+    <h:body>
+        
+        <h1>Fizzit</h1>
+        
+        <h:panelGroup class="trade-container" layout="block">
+            
+            <h:form class="barcode-form" id="form">
+                <h:inputText id="bcode"
+                             pt:placeholder="Enter a barcode"
+                             class="barcode-input"
+                             maxlength="13"
+                             pt:title="Enter a barcode"
+                             required="true"
+                             value="#{localTradeList.barcode}">
+                    <f:validator binding="#{barcodeValidator}"/>
+                    <f:validator binding="#{tradeLimitValidator}"/>
+                </h:inputText> 
+                <h:commandButton id="trade-btn"
+                                 class="barcode-input-btn active" 
+                                 value="Confirm"
+                                 action="#{localTradeList.submit}">
+                    <!--ajax for async update of table -->
+                    <f:ajax execute="@form" resetValues="true" render="@form trade-list" />
+                </h:commandButton>
+                <h:message id="trade-err" class="error" for="trade-btn"/>
+                <h:message id="errmsg" class="error" for="bcode"/>
+				<h:outputText id ="totalPrice" value = "#{localTradeList.price}"/>

```

Comment:
Small nitpick - Can you format the <h:outputText> element so it has the correct indentation.